### PR TITLE
Docs/frontend

### PR DESCRIPTION
**Pull request by luwol03**

### CODE REVIEW

File: docs/vocascan-frontend/installation/traefik.md [hunk 1]
```diff
@@ -1,3 +1,53 @@
...
+  tty: true
+  environment:
+    VOCASCAN_BASE_URL: "https://your-api-domain.com"
+  labels:
+    - "traefik.http.routers.frontend.rule=Host(`your-domain.com`)"
+    - "traefik.http.routers.frontend.tls=true"
+    - "traefik.http.routers.frontend.tls.certresolver=lets-encrypt"
+    - "traefik.http.services.frontend.loadbalancer.server.port=80"
+  networks:
+    - internal
```
Comment (luwol03) [inline docs/vocascan-frontend/installation/traefik.md]: why do you added internal?

Comment (noctera) [reply docs/vocascan-frontend/installation/traefik.md]: I made those configs after tutorials. In those you give everything, that you put everything that should available from outside in the `internal` and `web` network

Comment (luwol03) [reply docs/vocascan-frontend/installation/traefik.md]: Ok. But can you please remove the internal network (and also in production) as the nginx server doesnt neet to connect to the database. Thats just a security issue.

Comment (noctera) [reply docs/vocascan-frontend/installation/traefik.md]: I need the internal network for the server to find the database. Shouldn't be a security issue, as `external` is set to `false`

Comment (luwol03) [reply docs/vocascan-frontend/installation/traefik.md]: Yeah, but you don't need the internet network on the Frontend. That's my point.

Comment (noctera) [reply docs/vocascan-frontend/installation/traefik.md]: You are right. I just removed it

**Review by noctera — APPROVED** [review]

**Pull request merged**